scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1055,34 +1101,23 @@
-                                                          (map cached-queries/job-ent->user)
-                                                          frequencies)
-              user->number-total-considerable-jobs (->> considerable-jobs
-                                                        (map cached-queries/job-ent->user)
-                                                        frequencies)
+
               number-matched-jobs (count matched-job-uuids)
               number-considerable-jobs (count considerable-jobs)
               number-unmatched-jobs (- number-considerable-jobs number-matched-jobs)]
 
-          (log/info "In" pool-name "pool, matching offers to considerable jobs"
-                    {:jobs-considerable {:head-matched? matched-considerable-jobs-head?
-                                         :head-resources first-considerable-job-resources
-                                         :number-matched-jobs number-matched-jobs
-                                         :number-considerable-jobs number-considerable-jobs
-                                         :number-unmatched-jobs number-unmatched-jobs
-                                         :stats (jobs->stats considerable-jobs)
-                                         :user->number-matched user->number-matched-considerable-jobs
-                                         :user->number-total user->number-total-considerable-jobs
-                                         :user->number-unmatched (merge-with
-                                                                   -
-                                                                   user->number-total-considerable-jobs
-                                                                   user->number-matched-considerable-jobs)}
-                     :offers {:number-matched (count offers-scheduled)
-                              :number-total (count offers)
-                              :number-unmatched (- (count offers) (count offers-scheduled))
-                              :stats (offers->stats offers)}})
+          (handle-match-cycle-metrics {:considerable-jobs considerable-jobs
+                                       :head-matched? matched-considerable-jobs-head?
+                                       :head-resources first-considerable-job-resources
+                                       :matches matches
+                                       :max-considerable num-considerable
+                                       :number-considerable-jobs number-considerable-jobs
+                                       :number-matched-jobs number-matched-jobs
+                                       :number-unmatched-jobs number-unmatched-jobs
+                                       :offers offers
+                                       :offers-scheduled offers-scheduled
+                                       :pool-name pool-name
+                                       })
```

Comment:
A code style norm is to always put any closing punctuation at the end of the prior line. I.e., no line has closing punctuation on it by itself.

(The only exception when the prior line has a comment.)

Reply:
ah, this is a result of my testing. forgot to bring the closing punctuation back after.